src/python/py/models/builder.py

```diff
@@ -2813,6 +2874,12 @@ def get_args():
         help="Precision of model",
     )
 
+    parser.add_argument(
+        "--no_contrib_ops",
```

Comment:
I agree that this should be changed to `use_qdq` instead. We should also add this via the `extra_options` instead.